### PR TITLE
New Presence :yayy:

### PR DESCRIPTION
**Pull request by wwmon**
I make a new presence ❤️

### CODE REVIEW

**Comment by wwmon** [comment]
image
grar1

**Comment by wwmon** [comment]
wot

File: MyBOT/dist/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "author": {
-        "name": "mon#0010",
-        "id": "406184197405802497"
+        "name": "mon#0010 & Deivid#0045",
```
Comment (frozaum) [inline MyBOT/dist/metadata.json]: You should put your name (without the #) and your profile id that you can find by turning on the Developer Mode on Discord

**Comment by wwmon** [comment]
ok already

File: MyBOT/dist/metadata.json (hunk 1 above)
Comment (wwmon) [reply MyBOT/dist/metadata.json]: alweady

**Pull request merged**